### PR TITLE
[Refactor]: Add uniformity in export of util functions

### PR DESCRIPTION
**Pull request by pathange-s**
Code refactoring | Clean up and make util function exports uniform

### CODE REVIEW

**Comment by shalithasuranga** [comment]
Hello. @pathange-s  Thanks for sending this PR. Could you please check CI errors :)

**Comment by pathange-s** [comment]
@shalithasuranga  awaiting for approval to run workflows

**Comment by pathange-s** [comment]
@shalithasuranga  CI errors cleared !

File: src/utils.js
Comment (shalithasuranga) [inline src/utils.js:6]: Mm.. seems like a circular dependency here.. Not sure whether some Node.js versions display warnings on console about this import. Could you please check further about this and check whether there is another way to organize these functions?

**Review by Jassi10000 — COMMENTED** [review]
this change looks somewhat better than the old one 
but the only thing am not able to understand is , that what will you be getting in the console exactly
I think the console would be displying this : 

<code>figlet.textSync('Neutralinojs')</code>

Am sure that you are not confusing it with the console showing 
<code> Neutralinojs </code>

What are your views on this @shalithasuranga  ?

**Comment by pathange-s** [comment]
@Jassi10000 `console.log(function ('params'));` prints the returned value of the function. Also, you could have checked out to my branch, verified it and commented definitely.

Check [this](https://stackoverflow.com/a/24798809)  out for your understanding.

**Comment by Jassi10000** [comment]
> @Jassi10000 `console.log(function ('params'));` prints the returned value of the function. Also, you could have checked out to my branch, verified it and commented definitely.
> 
> Check [this](https://stackoverflow.com/a/24798809) out for your understanding.

Ahh got it buddy !!!
Anyways , I like your latest approach , that's why I wrote that it's better than previous one

**Comment by pathange-s** [comment]
@Jassi10000 ah, thanks !

File: src/utils.js
Comment (shalithasuranga) [inline src/utils.js:21]: Thanks for the updates..Now this code line seems duplicated I think. Also, I think the `checkCurrentProject` function will fail since `error(args)` is not defined. Do you think it's better to keep the source file in the original version? If we really need to make everything consistent in this file, as I understand, I think the only solution is to module.export all at the end of file, like this

```js
function a(){}
function b(){}
//...
//..
module.exports = {a, b};

```

Comment (pathange-s) [reply src/utils.js:21]: Umm. Few were exported directly and few were defined and exported. So, I just thought to change it to one format. You please let me know what to do. This is just a refactoring PR so anything works for me.

Comment (pathange-s) [reply src/utils.js:21]: @shalithasuranga what say, what do ? lets put it in one way rather than few direct exports and few implement + exports.

**Review by Dhananjay-JSR — APPROVED** [review]

File: src/utils.js
Comment (pathange-s) [reply src/utils.js:21]: @shalithasuranga ping

Comment (shalithasuranga) [reply src/utils.js:21]: Hey @pathange-s Thanks for reminding this again :) As I mentioned the current issue is the duplicated `textSync` call and [`error()`](https://github.com/neutralinojs/neutralinojs-cli/blob/908fca7b64ce60ee2e5ae3ff34209526cd9da1d3/src/utils.js#L26) function call (I will it will throw an error since the error function is not defined even though it's exported as `module.exports.error`).  

I think the original version is also okay. But, since you already submitted this change, I am suggesting you organize functions in a different way. The problem is that if we export a function with `module.exports.func`, we have to call it with either `module.exports.func()` or `this.func()` within the same file. Therefore I am suggesting you export all required functions at the end as I explained in the previous code snippet. Then, we can simply call `func()` within the same file and we can `require` them as usual.

Please let me know if you have questions or ideas. Thanks

Comment (pathange-s) [reply src/utils.js:21]: Gotcha ! Will do. Actually, defining all, then exporting it lgtm.

Comment (pathange-s) [reply src/utils.js:21]: Having function name as `figlet` and importing `figlet` (using require) is causing reference errors. 
[this](https://github.com/neutralinojs/neutralinojs-cli/blob/2611f36de9e94d47f6999aa1f0f86d295f19cfdb/src/utils.js#L3) and [this](https://github.com/neutralinojs/neutralinojs-cli/blob/2611f36de9e94d47f6999aa1f0f86d295f19cfdb/src/utils.js#L21)

Comment (pathange-s) [reply src/utils.js:21]:
```
/home/sainath/gsoc/njs/neutralinojs-cli/src/utils.js:20
let figlet = () => {
    ^

SyntaxError: Identifier 'figlet' has already been declared
```

Comment (shalithasuranga) [reply src/utils.js:21]: Ah, maybe `showArt` ?

Comment (pathange-s) [reply src/utils.js]: @shalithasuranga updated it as required. Check it out.

File: src/utils.js
Comment (shalithasuranga) [inline src/utils.js:46]: Could you please use one `module.export` for this? All other things look great and ready to be merged :tada:

Eg:
```js
module.exports = {
    error,
    isNeutralinojsProject,
   ... 
 ... 
}
```

Comment (pathange-s) [reply src/utils.js]: Ahh, how did I miss that. Doing it right away.

**Review by Dhananjay-JSR — APPROVED** [review]

**Review by shalithasuranga — APPROVED** [review]
All looks great now. Thank you very much for the contribution :+1: , Would you like to display your name in in the [committers list](https://neutralino.js.org/docs/contributing/committers) @pathange-s ?

We usually update this list with developers who contribute Neutralinojs often

**Comment by pathange-s** [comment]
@shalithasuranga I'm so grateful.  Add me to the list 😜

**Pull request merged**
